### DISCUSSION TITLE
Infinite loading on branch/yaml

### DISCUSSION THREAD

**Discussion by ferdiangunawan** [codemagic-ci-cd/codemagic-docs#1669] [Q&A] [answered]
I just got infinite loading when I want to run my workflow. Any help please?
<img width="637" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/49526345/47162749-dd13-4dfd-9ec2-933fce68c1b3">

**Comment by thedavidpeto** [comment]
Same here

**Reply by TFJL** [reply]
Yip same here

**Reply by ferdiangunawan** [reply]
I notice that Codemagic always get this problem since I used them in the early day. This is my third team that uses Codemagic and pays them well. we're just thinking back to using this service because of this infinite loading. hmm

**Reply by icarusdust** [reply]
Hi @TFJL @ferdiangunawan, should be alright now, can you please refresh the page and try again. Let me know how it goes.

**Reply by thedavidpeto** [reply]
As I see it does not work

**Reply by tacksman** [reply]
we happen same it.

**Reply by scotch-formitable** [reply]
Still not working for us unfortunately

**Reply by TFJL** [reply]
@icarusdust Problem still happens on my side unfortunately.

**Reply by tacksman** [reply]
Other team members are still unable to select branches as well.

**Reply by dtrdic** [reply, marked as answer]
should work now, please refresh the page and try again

**Reply by ferdiangunawan** [reply]
yes. thanks @dtrdic

**Comment by ferdiangunawan** [comment]
now it's working.
